docs/source/tutorials/configurable.rst

```diff
@@ -0,0 +1,108 @@
+==============================
+Configuration and plugin setup
+==============================
+
+Maybe you :doc:`played with commands </tutorials/playing-command>` for your
+plugin and now you want to make your plugin configurable. If you run an
+instance of Sopel yourself, you probably had to open and edit its configuration
+file.
+
+Usually located in your home directory, under the ``.sopel/`` folder, the
+configuration file is an INI file with sections defined by Sopel's core and by
+plugins. In this tutorial, let's see how to declare and use a configuration
+section dedicated to your plugin.
+
+
+Declaring your configuration
+============================
+
+To declare a configuration section, you must first create a subclass of
+:class:`~sopel.config.types.StaticSection`, and define attributes::
+
+    from sopel.config import types
+
+    class MyPluginSection(types.StaticSection):
+        fruits = types.ListAttribute('fruits')
+
+
+Telling Sopel about it
+======================
+
+Now, having a class in your plugin doesn't achieve much: you need to tell the
+bot about it. The best place to do so is in the :func:`setup` function
+hook of your plugin::
+
+    def setup(bot):
+        bot.settings.define_section('myplugin', MyPluginSection)
+
+This way, you tell Sopel that the ``[myplugin]`` section is used by your
+plugin, and to parse this section Sopel must use your class, i.e.
+``MyPluginSection``.
+
+
+Using your section
+==================
+
+In your configuration file, you can add the following lines:
+
+.. code-block:: ini
+
+    [myplugin]
+    fruits =
+        banana
+        apple
+        peach
+        strawberry
+
+And Sopel will expose that for you through ``bot.settings``. For example, you
+can write this command::
```

Comment:
Slight rewording, emphasize that the section is stored as an instance of the class and use its "full name" in this context (rather than referring to the entire configuration stored in `settings`).

```suggestion
And Sopel will store this section's options in ``bot.settings.myplugin`` using
an instance of the class defined earlier. These settings are now available for
your plugin to use! For example, you can write this command::
```

Reply:
Ditto, too much details. I'll use `bot.settings.myplugin` tho, that's better.